packages/app-store/office365calendar/lib/CalendarService.ts

```diff
@@ -21,11 +21,33 @@ import { O365AuthCredentials } from "../types/Office365Calendar";
 let client_id = "";
 let client_secret = "";
 
+interface IRequest {
+  method: string;
+  url: string;
+  id: number;
+}
+
+interface ISettledResponse {
+  id: string;
+  status: number;
+  headers: {
+    "Retry-After": string;
+    "Content-Type": string;
+  };
+  body: Record<string, any>;
+}
+
+interface IBatchResponse {
+  responses: ISettledResponse[];
+}
+
 export default class Office365CalendarService implements Calendar {
   private url = "";
   private integrationName = "";
   private log: typeof logger;
+  private accessToken: string | null = null;
```

Comment:
Made this a private value so it can be access while using any private function without passing it as parameter.